modules/meeting/app/controllers/recurring_meetings_controller.rb

```diff
@@ -130,6 +132,13 @@ def update
     end
   end
 
+  def delete_dialog
+    respond_with_dialog RecurringMeetings::DeleteDialogComponent.new(
+      meeting: @recurring_meeting,
+      project: @project
```

Comment:
(I based my `delete_dialog` controller action on `details_dialog`) ❓  Not sure we need to pass in project here? - see other comments on nested (project scoped) routes.

Reply:
If you don't need to create routes depending on the current project context, then you can omit  `@project`.